src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  --main-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  min-width: 1024px;
+}
+
+header {
+  width: calc(100% - 100px);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-transform: uppercase;
+  padding: 0 50px;
+  overflow: hidden;
+}
+
+
+.logo-link {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+
+nav ul {
```

Comment:
add class to `ul` tag and use it for styling

don't use tag selectors to add specific styles and don't use combined selectors without necessary